tests/model_serving/model_runtime/vllm/cpu/cpu_x86/utils.py

```
@@ -0,0 +1,90 @@
+    prompt: str,
+    max_tokens: int,
+) -> dict[str, Any]:
+    """Send a POST request to /v1/completions matching the OpenAI completions API."""
+    url = f"{get_exposed_isvc_url(isvc=isvc)}{OpenAIEnpoints.COMPLETIONS}"
+    payload = {
+        "model": isvc.instance.metadata.name,
+        "prompt": prompt,
+        "max_tokens": max_tokens,
+    }
+    LOGGER.info("Sending completions request", url=url, payload=payload)
+    response = requests.post(url, headers=RestHeader.HEADERS, json=payload, verify=False)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)